ARSIS-Unity/Assets/Scripts/EventManager/EventTypes/BaseArsisEvent.cs

```diff
@@ -19,6 +19,7 @@ public class BaseArsisEvent
 
         public string type;
         public long time;
```

Comment:
"time" gets changed to "label" in the MCP code but its still retained here. Also does not handle "label" key in the payload.

Reply:
The HMD and LMCC parts of this are incomplete, I can try and update those, but this is mostly just to get some MCP code that can be used for devleoping the client systems

Reply:
You wont need to update HMD and the LMCC parts, I can update them later on. I don't wanna pollute this issue by incorporating non-MCP related things. Thank you! 🙏